src/scss/components/_icons.scss

```diff
@@ -0,0 +1,47 @@
+.bi{
+    vertical-align: -.125em;
```

Comment:
This looks out of place. What's the intention?

Reply:
Used it to get icons centered with the text. Have removed it and used classes on each icon in my HTML instead `d-flex align-items-center`